docs/documentation_guide.md

```diff
@@ -17,8 +17,9 @@ Learn how to use Fondant to build your own data processing pipeline.
 
 -> Design your own fondant [pipeline](pipeline.md) using the Fondant pipeline SDK.  
 -> Use existing [reusable components](components/hub.md) to build your pipeline.
--> Build your own [custom component](components/custom_component.md) using the Fondant component
-SDK.  
+-> Build your own custom [python component](components/custom_containerized_component.md) 
+or [containerized component](components/custom_containerized_component.md) using the Fondant component
```

Comment:
Maybe we can use something like `share components`. Then the story would be, build your custom component with the python component and later on, if you want to share/publish it, the more "complex" documentation how to package the component into a reusable one.

Reply:
This makes sense!